src/entities/Role.ts

```diff
@@ -0,0 +1,29 @@
+import { Entity, Column, BaseEntity, PrimaryGeneratedColumn, OneToMany} from "typeorm";
+import { Client } from "./Client";
+
+export enum RoleTypes{
+    ADMIN = 'admin',
```

Comment:
also depend on constants for example 
ADMIN = Constants.ADMIN 
- to allow anyone in the project to use the same type as it is without changing its string or use it with a wrong string with a typo for example